files/preseeds/openSUSE-Leap.autoyast

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <general>
+    <mode>
+      <confirm config:type="boolean">false</confirm>
+    </mode>
+  </general>
+  <networking>
+    <keep_install_network config:type="boolean">true</keep_install_network>
+  </networking>
+  <partitioning  config:type="list">
+    <drive>
+      <device>/dev/sda</device>
+      <use>all</use>
+    </drive>
+  </partitioning>
+  <software>
+    <install_recommended config:type="boolean">true</install_recommended>
+    <patterns config:type="list">
+      <pattern>base</pattern>
+      <pattern>console</pattern>
+    </patterns>
+    <packages config:type="list">
+      <package>perf</package>
+      <package>zypper-aptitude</package>
+    </packages>
+  </software>
+  <users config:type="list">
+    <user>
+      <username>root</username>
+      <uid>0</uid>
+      <gid>0</gid>
+      <encrypted config:type="boolean">false</encrypted>
+      <fullname>Root User</fullname>
+       {% for key in ssh_keys %} echo "<listentry>{{key}}</listentry>\n" {% endfor %}
```

Comment:
I need to investigate the ssh key (string) handling in MrP but in the meantime, since we store ssh keys with \n delimiting them for each user (if that user has multiple ssh keys), handling in AutoYast's case (this is not being interpreted by the installer) has to be done in the jinja.

```python
  <users config:type="list">
    <user>
      <username>root</username>
<!-- From Leap 15.1+, password based ssh root login is disabled -->
      <user_password>linux</user_password>
      <uid>0</uid>
      <gid>0</gid>
      <encrypted config:type="boolean">false</encrypted>
      <fullname>Root User</fullname>
      {% for key in ssh_keys %}
      {% set keys = key.split("\\n") %}
      {% for k in keys %}
      <listentry>{{k}}</listentry>
      {% endfor %}
      {% endfor %}
    </user>
  </users>
```

Reply:
I doubt this is needed, as it should not be different from debian/centos.
Let's test both solutions when Mr-P PR get merged and deployed.

Reply:
Indeed, it does not interpret '\n' and just print '\n'. PR updated accordingly. Thanks. :)